spec/datadog/core/crashtracking/component_spec.rb

```diff
@@ -329,13 +377,14 @@
 
           Datadog.configure do |c|
             c.agent.host = 'google.com'
+            c.agent.port = '12345'
```

Comment:
```suggestion
            c.agent.port = 12345
```

Reply:
I also spotted that it's documented as being a string (https://github.com/DataDog/dd-trace-rb/blob/bc841812cbc73fdba7b798bfcf74140d6df3da85/lib/datadog/core/configuration/settings.rb#L63) although the agent settings resolver accepts both ints and strings.

I'll apply the suggestion anyway :)